returnn/tf/layers/basic.py

```diff
@@ -1179,9 +1179,7 @@ def __init__(self, position, **kwargs):
     if position.batch_dim_axis is not None:
       position = position.copy_as_batch_major()
     else:
-      position = position.copy_add_batch_dim(batch_dim_axis=0)
-      position.placeholder = tf.tile(
-        position.placeholder, [tf.shape(x.placeholder)[0]] + [1] * (position.batch_ndim - 1))
+      position = position.copy_add_batch_dim(batch_dim_axis=0, batch_dim=tf.shape(x.placeholder)[0])
```

Comment:
```suggestion
      position = position.copy_add_batch_dim(batch_dim_axis=0, batch_dim=tf.shape(x.placeholder)[position.batch_dim_axis])
```
?

Reply:
Hm yes that is nicer (but with `x.batch_dim_axis`). I just copied it from how it was before, and it shouldn't matter because `x` is copied as batch major anyway. But still, yup!

Reply:
Hm yes that is nicer (but with `x.batch_dim_axis`). I just copied it from how it was before, and it shouldn't matter because `x` is copied as batch major anyway. But still, yup!